CMakeLists.txt

```diff
@@ -194,10 +194,6 @@ endif ()
 set(ENTERPRISE_INCLUDE_DIR "enterprise")
 option(USE_ENTERPRISE "enable enterprise build" OFF)
 
-# we want the following definitions to be in effect for both rocksdb and arangodb
-add_definitions("-DNROCKSDB_THREAD_STATUS")
-add_definitions("-DROCKSDB_SUPPORT_THREAD_LOCAL")
```

Comment:
note: the new version of RocksDB does not react to a define `ROCKSDB_SUPPORT_THREAD_LOCAL` anymore, so the line can be removed.